pages/apis/graphql/graphql_cookbook.md

```diff
@@ -540,7 +540,80 @@ mutation {
   }
 }
 ```
+## Clusters
 
+A collection of common tasks with clusters using the GraphQL API
+
+### Create a queue
+
+Creates a new queue in a cluster
+
+```graphql
+mutation {
+  clusterQueueCreate(input: {
+    organizationId: "T3JnYW5pemF0aW9uLS0tZDZmNzU1NTEtZTM2OC00NzNmLTk1YzctNmFlOWU1OTNkOWU2",
+    clusterId: "Q2x1c3Rlci0tLWU3YWFjN2Q3LTg2ZmYtNDJjZi04NzYyLTM0ZjAwY2Q5MWRkOQ=="
+    key: "Default",
+    description: "The default queue"
```

Comment:
Make it clearer that these values need to be replaced:

```suggestion
    organizationId: "org-id",
    clusterId: "cluster-id"
    key: "queue-key",
    description: "queue-description"
```

^ Applies to each entry.